ch_backup/logic/lock_manager.py

```diff
@@ -106,7 +108,23 @@ def _zk_flock(self) -> None:
             if not client.exists(self._process_zk_lockfile_path):
                 client.create(self._process_zk_lockfile_path, makepath=True)
             self._zk_lock = client.Lock(self._process_zk_lockfile_path)
-            if not self._zk_lock.acquire(blocking=False):
-                sys.exit(self._exitcode)
+            force_exit = self._exitcode == 0
```

Comment:
Semantics of using `exitcode` config field was changed by this. Originally it was exit code in case of failed zk lock acquisition. But now only `0` will be actual exit code. 
As for exception, it can be catched and swallowed (maybe unintentionally).
Is it intended ?